jenkins-scripts/dsl/ignition_collection.dsl

```diff
@@ -18,22 +18,22 @@ ignition_collections = [
     // These are the branches currently targeted at the upcoming collection
     // They're in topological order
     nightly_jobs: [
-          'tools'     : [ debbuild: 'ign-tools2'      , branch: 'main' ],
-          'cmake'     : [ debbuild: 'ign-cmake3'      , branch: 'main' ],
-          'utils'     : [ debbuild: 'ign-utils2'      , branch: 'main' ],
-          'math'      : [ debbuild: 'ign-math7'       , branch: 'main' ],
-          'plugin'    : [ debbuild: 'ign-plugin2'     , branch: 'main' ],
-          'common'    : [ debbuild: 'ign-common5'     , branch: 'main' ],
-          'msgs'      : [ debbuild: 'ign-msgs9'       , branch: 'main' ],
-          'rendering' : [ debbuild: 'ign-rendering7'  , branch: 'main' ],
-          'sdformat'  : [ debbuild: 'sdformat13'      , branch: 'main' ],
-          'fuel-tools': [ debbuild: 'ign-fuel-tools8' , branch: 'main' ],
-          'transport' : [ debbuild: 'ign-transport12' , branch: 'main' ],
-          'gui'       : [ debbuild: 'ign-gui7'        , branch: 'main' ],
-          'sensors'   : [ debbuild: 'ign-sensors7'    , branch: 'main' ],
-          'physics'   : [ debbuild: 'ign-physics6'    , branch: 'main' ],
-          'gazebo'    : [ debbuild: 'ign-gazebo7'     , branch: 'main' ],
-          'launch'    : [ debbuild: 'ign-launch6'     , branch: 'main' ],
+          'cmake'     : [ debbuild: 'ign-cmake3'      , branch: 'gz-cmake3' ],
+          'tools'     : [ debbuild: 'ign-tools2'      , branch: 'gz-tools2' ],
+          'utils'     : [ debbuild: 'ign-utils2'      , branch: 'gz-utils2' ],
+          'math'      : [ debbuild: 'ign-math7'       , branch: 'gz-math7' ],
+          'plugin'    : [ debbuild: 'ign-plugin2'     , branch: 'gz-plugin2' ],
+          'common'    : [ debbuild: 'ign-common5'     , branch: 'gz-common5' ],
+          'msgs'      : [ debbuild: 'ign-msgs9'       , branch: 'gz-msgs9' ],
+          'rendering' : [ debbuild: 'ign-rendering7'  , branch: 'gz-rendering7' ],
+          'sdformat'  : [ debbuild: 'sdformat13'      , branch: 'sdf13' ],
+          'fuel-tools': [ debbuild: 'ign-fuel-tools8' , branch: 'gz-fuel-tools8' ],
+          'transport' : [ debbuild: 'ign-transport12' , branch: 'gz-transport12' ],
+          'gui'       : [ debbuild: 'ign-gui7'        , branch: 'gz-gui7' ],
+          'sensors'   : [ debbuild: 'ign-sensors7'    , branch: 'gz-sensors7' ],
+          'physics'   : [ debbuild: 'ign-physics6'    , branch: 'gz-physics6' ],
+          'gazebo'    : [ debbuild: 'ign-gazebo7'     , branch: 'gz-sim7' ],
```

Comment:
This branch doesn't exist yet, we're waiting for the forward-port.

Reply:
https://github.com/gazebo-tooling/release-tools/pull/785/commits/69934c4b4e7a5b58cf2b5443bbf628a628b8ff1b